quickevent/app/plugins/Event/src/eventplugin.h

```diff
@@ -83,6 +83,7 @@ class EVENTPLUGIN_DECL_EXPORT EventPlugin : public qf::qmlwidgets::framework::Pl
 	Q_SIGNAL void sqlServerConnectedChanged(bool is_open);
 
 	Q_SIGNAL void eventOpened(const QString &event_name);
+	Q_SIGNAL void eventClosed();
```

Comment:
we have eventOpenChanged already, there is no need for eventClosed signal. Connect to eventOpenChanged(false) instead. If you wand to reduce number of lambdas in `connect`, keep `Q_SIGNAL eventClosed()` and connect eventOpenChanged to `[this](bool b) {if(!b) emit eventClosed();}`